src/pattern-library/components/patterns/ColorFoundations.js

```diff
@@ -1,75 +1,62 @@
+import classnames from 'classnames';
 import Library from '../Library';
 
-// TODO:
-// - color descriptions and guidance
-// - hex and other metadata about colors
-// - foreground color examples
-// - more examples of overriding with `!` classes
-// - valid contrast combinations
-
-const brand = ['brand'];
-
-const greys = [
-  'white',
-  'grey-0',
-  'grey-1',
-  'grey-2',
-  'grey-3',
-  'grey-4',
-  'grey-5',
-  'grey-6',
-  'grey-7',
-  'grey-8',
-  'grey-9',
-];
-
-const states = ['success', 'notice', 'error'];
-
-function ColorSwatch({ colorName }) {
+function ColorSwatch({ colorClass, colorName }) {
   return (
     <div>
-      <div
-        className={`hyp-u-bg-color--${colorName}`}
-        style="width:16rem;height:2rem;margin-right:1em"
-      />
+      <div className={classnames(colorClass, 'w-64 h-8 mr-4')} />
       <p>
         <i>{colorName}</i>
       </p>
     </div>
   );
 }
 
-const brandExamples = brand.map(colorName => (
-  <ColorSwatch key={colorName} colorName={colorName} />
-));
-
-const greyExamples = greys.map(colorName => (
-  <ColorSwatch key={colorName} colorName={colorName} />
-));
-
-const stateExamples = states.map(colorName => (
-  <ColorSwatch key={colorName} colorName={colorName} />
-));
+const brandExamples = (
+  <>
+    <ColorSwatch colorClass="bg-brand" colorName="brand" />
+    <ColorSwatch colorClass="bg-brand-dark" colorName="brand-dark" />
+  </>
+);
+
+const greyExamples = (
+  <>
+    <ColorSwatch colorClass="bg-white" colorName="white" />
```

Comment:
A reminder that this can't be looped/scripted as the utility styles need to be present as strings in the source, or else tailwind's purge won't see them and they won't be included. So each classname does need to be included here. I can think of other ways to do this — the string just literally needs to be present in this source, even in comments — but this seemed the least mysterious, if not so DRY.